course/admin.py

```diff
@@ -6,8 +6,32 @@
 from course.models.models import Question, VariableQuestion, MultipleChoiceQuestion, Submission, QuestionCategory, \
     CheckboxQuestion, JavaSubmission, JavaQuestion, TokenValue, MultipleChoiceSubmission, UserQuestionJunction
 from course.models.parsons_question import ParsonsQuestion, ParsonsSubmission
+import copy  # Python native copy functionality
 
 
+def duplicate_question(modeladmin, request, queryset):
+    '''
+    Django Admin Action that will duplicate selected question instances and set their author to the currently logged-in user
+    '''
+    # qitem refers to the instances of the selected questions
+    for qitem in queryset:
+        dupe_q = copy.deepcopy(qitem)
```

Comment:
here you can use qitem.clone().
Also qitem and dupe_q are bad names try thinking of a better name using python name conventions.
https://realpython.com/python-pep8/#how-to-choose-names